app/controllers/events_controller.rb

```diff
@@ -214,8 +214,9 @@ def create
   end
 
   def update
-    event_id = params[:id]
+    return copy if is_a_copy?
```

Comment:
`copy` is not returning anything but redirecting. I think of `copy and return if ...` in this case being a better practice. Am I right?

Reply:
yes, you're right. should we do it this way?
```
  if is_a_copy?
    copy
    return
  end
```

Reply:
Wouldn't this be the same as mine?

We have three versions now to discuss:

1. the old one
```
return copy if is_a_copy?
```

2. my suggestion
```
copy and return if is_a_copy?
```

3. your suggestions
```
if is_a_copy?
  copy
  return
end
```

Where are the differences in your eyes?

Reply:
> Wouldn't this be the same as mine?
Yes you are right.  I didn't think of `copy and return if` as a pattern.